server/clcachesrv.py

```diff
@@ -5,14 +5,18 @@
 import os
 import pickle
 import signal
+import argparse
+import re
 
 import pyuv
 
 class HashCache(object):
-    def __init__(self, loop):
+    def __init__(self, loop, excludePatterns, disableWatching):
         self._loop = loop
         self._watchedDirectories = {}
         self._handlers = []
+        self._excludePatterns = excludePatterns if excludePatterns else []
```

Comment:
Just a minor remark - this expression can be shortened to `excludePatterns or []`.